imitation/scripts/config/data_collect.py

```diff
@@ -8,10 +8,20 @@
 
 @data_collect_ex.config
 def data_collect_defaults():
-    total_timesteps = int(1e6)
-    num_vec = 8
+    env_name = "FILL ME"  # The gym.Env name
+    total_timesteps = int(1e6)  # Number of training timesteps in model.learn()
+    num_vec = 8  # Number of environments in DummyVecEnv
     make_blank_policy_kwargs = DEFAULT_BLANK_POLICY_KWARGS
 
+    rollout_save = True  # Whether to save rollout files.
+    rollout_save_interval = 50  # The number of training updates between saves.
+    rollout_save_n_samples = 2000  # The number of timesteps saved per file.
+    rollout_dir = "data/rollouts"  # The directory that rollouts are saved in.
```

Comment:
We probably don't care about Windows compatibility, but if we did then we should use `os.path.join("data", "rollouts")` or `os.path.normpath("data/rollouts")`